assets/css/v2/pre_fare/body_right/surge.scss

```diff
@@ -0,0 +1,25 @@
+// body total height: 1648px
+
+.body-right-surge {
+  position: relative;
+}
+
+.body-right-surge__upper {
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  width: 1080px;
+  height: 876px;
+  overflow: hidden;
+}
+
+.body-right-surge__lower {
+  position: absolute;
+  top: 918px;
+  left: 0;
+
+  width: 1080px;
+  height: 876px;
```

Comment:
I think this height and that of the upper slot should add up to the body total height. Currently we have 876 + 876 = 1752 ≠ 1648 (assuming the comment at the top of the file is accurate)

Reply:
Ah that comment was not correct! The pre-fare screen body is 1720px tall.

Reply:
Yep. I originally copied this file from `normal.scss`, which also happened to be incorrect 😅 Got them both corrected.